GeoEntity.md

```diff
@@ -0,0 +1,280 @@
+#GeoEntity API
+
+&nbsp;
+
+#Contents
+
+- [Description](#description)
+- [All Countries Request](#all-countries-request)
+- [All Administrative Area Ones Request](#all-administrative-area-ones-request)
+- [All Administrative Area Twos Request](#all-administrative-area-twos-request)
+- [All Localities Request](#all-localities-request)
+- [All Postal Codes Request](#all-postal-codes-request)
+- [All Metropolitan Statistical Areas Request](#all-metropolitan-statistical-areas-request)
+- [Versioning](#versioning)
+
+&nbsp;
+----------------------------
+#Description
+
+This service provides aggregated geography entities. Currently the following requests could be made.
```

Comment:
also, you should put a bit more info in here about this service. mention that it uses the same underlying data as our geocoder. mention that we cannot guarantee 100% completeness, but the data should be fairly comprehensive. (maybe mention that we can add missing locations as needed.)